src/Symfony/Component/HttpClient/Response/ResponseTrait.php

```diff
@@ -49,7 +50,7 @@ trait ResponseTrait
         'error' => null,
     ];
 
-    private $multi;
```

Comment:
The trait itself does not use that property. And since both implementations use different classes for `$multi`, I've moved the property to the classes.